index.js

```diff
@@ -1430,8 +1430,7 @@ module.exports = class Autobase extends ReadyResource {
 
       // autobase version was bumped
       let upgraded = false
-      if (update.version > this.version) {
-        if (!this._onUpgrade(update.version)) return // failed
+      if (update.version > this.version && this._onUpgrade(update.version)) {
```

Comment:
is the change here intentional that it continues the function now if onUpgrade returns false vs before where it would return?

Reply:
its misleading, _onUpgrade either throws or returns true

Reply:
have updated to make this more clear